pkg/api/v1beta3/dynakube/oneagent_types.go

```diff
@@ -133,7 +133,7 @@ type AppInjectionSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Resource Requirements",order=15,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:resourceRequirements"}
 	InitResources *corev1.ResourceRequirements `json:"initResources,omitempty"`
 
-	// The OneAgent image that is used to inject into Pods.
+	// Use a custom OneAgent image to download binaries
```

Comment:
this sounds a bit too general to me

Reply:
I didn't update the wording too much, just tried to get the format in line with version/image fields.

In our docs we use the term "OneAgent CodeModules" so I'll go with that. Still not sure if "downloading binaries" is the correct short form though:
```suggestion
	// Use a custom OneAgent CodeModules image to download binaries
```